arduino/cores/packagemanager/package_manager.go

```diff
@@ -174,8 +191,10 @@ func (pm *PackageManager) ResolveFQBN(fqbn *cores.FQBN) (
 	// Find board
 	board := platformRelease.Boards[fqbn.BoardID]
 	if board == nil {
+		// Try looking for the closest match; if found, suggest it
+		suggestion := pm.findClosestMatchFqbn(fqbn)
 		return targetPackage, platformRelease, nil, nil, nil,
-			fmt.Errorf("board %s:%s not found", platformRelease, fqbn.BoardID)
+			fmt.Errorf("board %s:%s not found, did you mean %s?", platformRelease, fqbn.BoardID, suggestion)
```

Comment:
```suggestion
			fmt.Errorf("board %s not found, did you mean %s?", fqbn.StringWithoutConfig(), suggestion)
```

`platformRelease.String()` would also print the version of the platform so the error would be quite confusing.